www/source/partials/docs/_dev-pkgs-exports.html.md.erb

```diff
@@ -74,7 +74,7 @@ For an example of using Docker Compose to run multiple Habitat containers togeth
 9. Now, start your package using the hab binary which is included in the tar archive
 
     ```shell
-    $ sudo /hab/bin/hab start yourorigin/yourpackage
+    $ sudo /hab/bin/hab sup run yourorigin/yourpackage
```

Comment:
I can't find this anywhere on the active website. If it is live and still says `hab start`, it should be fixed as indicated here.